internal/commands/auth/print_access_token.go

```diff
@@ -28,13 +28,6 @@ func NewCmdPrintAccessToken(ctx *cmd.Context) *cmd.Command {
 				Preamble: "To print the access token:",
 				Command:  "$ hcp auth print-access-token",
 			},
-			{
-				Preamble: "To use the access token when curling an API:",
-				Command: heredoc.New(ctx.IO, heredoc.WithPreserveNewlines()).Must(`
```

Comment:
Does this get executed? It would be nice to keep it in the docs if possible.

Reply:
Sure, we can keep it. I was just thinking that if a user attempted this example with the docker container it wouldn't have `curl` installed, but I suppose it could just be manually installed. Will do as you suggest.